src/Uno.UI/UI/Xaml/Controls/MediaPlayerElement/MediaTransportControls.cs

```diff
@@ -152,143 +422,322 @@ protected override void OnApplyTemplate()
 
-			_skipForwardButton = this.GetTemplateChild(SkipForwardButtonName) as Button;
-			_skipForwardButton?.SetBinding(Button.VisibilityProperty, new Binding { Path = "IsSkipForwardButtonVisible", Source = this, Mode = BindingMode.OneWay, FallbackValue = Visibility.Collapsed, Converter = trueToVisible });
-			_skipForwardButton?.SetBinding(Button.IsEnabledProperty, new Binding { Path = "IsSkipForwardEnabled", Source = this, Mode = BindingMode.OneWay, FallbackValue = true });
+			// Release Closed Caption Selection button's event handlers
+			//IFC(DetachHandler(m_epCCSelectionButtonClickHandler, m_tpCCSelectionButton));
 
-			_nextTrackButton = this.GetTemplateChild(NextTrackButtonName) as Button;
-			_nextTrackButton?.SetBinding(Button.VisibilityProperty, new Binding { Path = "IsNextTrackButtonVisible", Source = this, Mode = BindingMode.OneWay, FallbackValue = Visibility.Collapsed, Converter = trueToVisible });
-			_nextTrackButton.Tapped -= NextTrackButtonTapped;
-			_nextTrackButton.Tapped += NextTrackButtonTapped;
+			// Release Play Rate Selection button's event handlers
+			UnhookButtonClick(m_tpPlaybackRateButton, PlaybackRateButtonTapped);
 
-			_previousTrackButton = this.GetTemplateChild(PreviousTrackButtonName) as Button;
-			_previousTrackButton?.SetBinding(Button.VisibilityProperty, new Binding { Path = "IsPreviousTrackButtonVisible", Source = this, Mode = BindingMode.OneWay, FallbackValue = Visibility.Collapsed, Converter = trueToVisible });
-			_previousTrackButton.Tapped -= PreviousTrackButtonTapped;
-			_previousTrackButton.Tapped += PreviousTrackButtonTapped;
+			// Release Video Mute/Volume button's event handlers
+			//IFC(DetachHandler(m_epVolumeButtonClickHandler, m_tpVideoVolumeButton));
 
-			_fastForwardButton = this.GetTemplateChild(FastForwardButtonName) as Button;
-			_fastForwardButton?.SetBinding(Button.VisibilityProperty, new Binding { Path = "IsFastForwardButtonVisible", Source = this, Mode = BindingMode.OneWay, FallbackValue = Visibility.Collapsed, Converter = trueToVisible });
-			_fastForwardButton?.SetBinding(Button.IsEnabledProperty, new Binding { Path = "IsFastForwardEnabled", Source = this, Mode = BindingMode.OneWay, FallbackValue = true });
+			// Release Audio Mute button's event handlers
+			UnhookButtonClick(m_tpMuteButton, ToggleMute);
 
-			_rewindButton = this.GetTemplateChild(RewindButtonName) as Button;
-			_rewindButton?.SetBinding(Button.VisibilityProperty, new Binding { Path = "IsFastRewindButtonVisible", Source = this, Mode = BindingMode.OneWay, FallbackValue = Visibility.Collapsed, Converter = trueToVisible });
-			_rewindButton?.SetBinding(Button.IsEnabledProperty, new Binding { Path = "IsFastRewindEnabled", Source = this, Mode = BindingMode.OneWay, FallbackValue = true });
+			// Release Full Window button's event handlers
+			//IFC(DetachHandler(m_epFullWindowButtonClickHandler, m_tpFullWindowButton));
```

Comment:
@Xiaoy312 Do we need to implement those?

Reply:
not sure which ones are you talking about?
- for Button Visibility+Enable property toggle, their implementation have changed to be done through this:
	```cs
	internal override void OnPropertyChanged2(DependencyPropertyChangedEventArgs args)
	{
		UpdateMediaControlState(args.Property);
	}
	```
- regarding releasing of certain events, i might have missed some... but most of hook up should be done, as i have tested each button